test/Dialect/Calyx/emit.mlir

```diff
@@ -102,6 +106,12 @@ calyx.program {
               calyx.if %c1.in with @Group2 {
                 calyx.enable @Group2
               }
+              calyx.if %c1.in {
+                calyx.enable @Group2
```

Comment:
Another restriction on `comb group` that I might have not mentioned: They cannot appear inside normal control operators like `seq` or `par`. The places where `comb group` and `group` may appear are disjoint.